src/sentry/tasks/relay.py

```diff
@@ -240,12 +312,51 @@ def compute_projectkey_config(key):
         return get_project_config(key.project, project_keys=[key], full_config=True).to_dict()
 
 
+def project_keys_to_update(public_key=None):
+    """Returns the project keys which need to have their config updated.
+
+    Queries the database for the required project keys.
+    """
+    from sentry.models import ProjectKey
+
+    if public_key is None:
+        return []
+
+    try:
+        return [ProjectKey.objects.get(public_key=public_key)]
+    except ProjectKey.DoesNotExist:
+        # In this particular case, where a project key got deleted and
+        # triggered an update, we know that key doesn't exist and we want to
+        # avoid creating more tasks for it.
+        projectconfig_cache.set_many({public_key: {"disabled": True}})
+        return []
+
+
+def compute_project_configs(project_keys):
+    """Computes the project configs for all given project keys."""
+    from sentry.models import ProjectKeyStatus
+    from sentry.relay.config import get_project_config
+
+    config_cache = {}
```

Comment:
this only ever used to compute a single config in reality, there is still a lot of baggage from where this function came.  In a few months time it'll be a puzzle while this is written this way and whether someone somehow still calls this with multiple keys and why is it than different from `compute_projectkey_config()`.